script/glam/run_glam_sql

```diff
@@ -60,7 +51,7 @@ function run_query {
         --max_rows=0 \
         --use_legacy_sql=false \
         $additional_arguments \
-        --project_id="$PROJECT" \
+        --project_id="$DEST_PROJECT" \
```

Comment:
What's the use case for setting a different project here? Testing?  Isn't it implied by the directory structure `sql/$PROJECT/` that the destination is `$PROJECT`?

Reply:
Yeah, it's primarily for testing. I think Marina set up a `glam-desktop-dev` project, which would mean the DST_PROJECT would need to be set appropriately.